.cirrus.yml

```diff
@@ -743,6 +743,38 @@ upgrade_test_task:
     always: *logs_artifacts
 
 
+image_build_task:
+    name: "Build multi-arch $CTXDIR"
+    alias: image_build
+    skip: *tags
+    # FIXME before merge: Only run nightly on 'main' branch
+    # only_if: $CIRRUS_CRON == 'multiarch'
+    only_if: $CIRRUS_CHANGE_TITLE =~ '.*CI:BUILD.*'  # For PR testing
+    depends_on:
+        - ext_svc_check
+    timeout_in: 120m  # emulation is sssllllooooowwww
+    gce_instance:
+        <<: *standardvm
+        image_name: build-push-${IMAGE_SUFFIX}
+        # More muscle required for parallel multi-arch build
+        type: "n2-standard-4"
+    env:
+        # FIXME before merge: Add encrypted push credentials
+        DRYRUN: 1
+    matrix:
+        - env:
+            CTXDIR: contrib/podmanimage/upstream
+        - env:
+            CTXDIR: contrib/podmanimage/testing
+        - env:
+            CTXDIR: contrib/podmanimage/stable
+        - env:
+            CTXDIR: contrib/hello
+    script:
+        - set -a; source /etc/automation_environment; set +a
+        - main.sh file://$PWD $CTXDIR
```

Comment:
Note to me: `file://$PWD` is wrong, that should say `https://github.com/containers/podman.git`.